doc/src/manual/methods.md

```diff
@@ -72,17 +72,17 @@ Applying it to any other types of arguments will result in a [`MethodError`](@re
 julia> f(2.0, 3)
 ERROR: MethodError: no method matching f(::Float64, ::Int64)
 Closest candidates are:
-  f(::Float64, !Matched::Float64) at none:1
+  f(::Float64, !Matched::Float64) at REPL[n]:1
```

Comment:
Why the change? This will now fail CI since those are tested there.

Reply:
oh my omission, thank @fredrikekre :)

Reply:
hi @fredrikekre ，i have check the failed [ci](https://travis-ci.org/JuliaLang/julia/jobs/425255392) again, it is a fault caused by REPL[n] indeed.
but i have no idea about why ci did different from illustration above.
functions related are all defined, "Closet candidates" should be "REPL[n]:1" and "Stacktrace" may be "none:0"; if fail into something "Undef~", there only "Stacktrace" which may be "none:0".

Reply:
sorry @fredrikekre :( i had seen i went into a incorrect branch as @garrison said.
mybe it is "none:1" in master and i am working at release-v1.0.0 branch.